include/install-packages.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+b() {
+	$DIR/../vdpm $1
+}
+
+install_packages() {
```

Comment:
Maintaining two identical lists is annoying, maybe do a `git checkout` on https://github.com/vitasdk/packages/blob/master/travis_packages.sh?

Reply:
We can install this in buildscripts, but then it won't work out of the `vdpm` checkout (I'm not sure what you mean by do a `git checkout` of it)
also, given https://github.com/vitasdk/packages/pull/28#issuecomment-309392506 we might want to build some stuff twice eventually, this shouldn't matter much now but I'm thinking how much the two lists will diverge eventually

Reply:
I would suggest to do 
`git archive --remote=ssh://git@github.com/vitasdk/packages.git HEAD travis_packages.sh | tar -x` but apparently GitHub doesn't support `git-archive`.

Maybe source `https://raw.githubusercontent.com/vitasdk/packages/master/travis_packages.sh`?